### PR TITLE
New 1.19.2 Mappings & Better Readme

### PR DESCRIPTION
**Pull request by vinc3nz**
- Mappings in net.minecraft.client.particles
- Other stuff

### CODE REVIEW

**Comment by vinc3nz** [comment]
Also updated the Readme a bit

File: README.md [hunk 1]
```diff
@@ -18,13 +23,20 @@ Parchment is an open community-sourced modloader-neutral set of mappings of para
...
 See the [`docs/CONTRIBUTING.md` file](docs/CONTRIBUTING.md) for information on contributing. Also look at the 
 [repository wiki](https://github.com/ParchmentMC/Parchment/wiki) for more information, such as links to the mapping 
 standards and PR review process documents.
 
+__**Contributors:**__ 
+
+<a href="https://github.com/parchmentmc/parchment/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=parchmentmc/parchment" />
+</a>
+
```
Comment (marchermans) [inline README.md]: I don't support having this in the Readme.md.

At least not added via PR without discussing this first with the team.

Please remove this.

**Review by marchermans — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vinc3nz) [reply README.md]: Removed

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
+<p align="center">
```
Comment (Lanse505) [inline README.md]: I'm going to be honest, I'm not a fan of this change being done in this PR.
We're already working on clarifications and updating our documentation.

Please remove this change to README.md for this PR.

**Review by Lanse505 — CHANGES_REQUESTED** [review]
Please remove the change to the README.md.
Otherwise this PR is fine.

**Comment by vinc3nz** [comment]
Removed

**Review by Lanse505 — APPROVED** [review]
LGTM

**Review by marchermans — APPROVED** [review]

**Pull request merged**